docs/upgrade/upgrade-7.17-8.x.asciidoc

```diff
@@ -0,0 +1,155 @@
+... If you use any watcher-based {kib} scheduled {kibana-ref}/reporting-getting-started.html[reporting], ensure that it is working properly.
+
+. Upgrade your ingest components (such as {ls}, {fleet} and {agent}, {beats}, etc.) Refer to the {stack-ref}/upgrading-elastic-stack.html[Elastic Stack upgrade docs] for details.
+
+. Validate Ingest is operating OK.
+.. Open *Discover*, go through data views for each of your expected ingest data streams, and ensure that data is being ingested in the expected format and volume. 
+
+. Validate that {elastic-sec} is operating OK.
+.. Ensure that your detection rules are re-enabled.
+.. Ensure that any SOAR workflows that consume alerts are working.
+.. Verify that any custom dashboards your team has created are working properly, especially if they operate on alert (signal) documents.
+
+. Confirm with your appropriate stakeholders that the upgrade process is complete.
+
+[float]
+=== Post-upgrade steps
+
+The following sections describe procedures to complete after upgrading {elastic-sec} to 8.x.
+
+[float]
+[[reenable-rules-upgrade]]
+==== Re-enable disabled rules
+
+Any active rules when you upgrade from 7.17 to 8.0.1 or newer are automatically disabled, and a tag named `auto_disabled_8.0` is added to those rules for tracking purposes. Once the upgrade is complete, you can filter rules by the newly added tag, then use bulk actions to re-enable them:
+
+. Go to the Rules page (*Detect -> Rules*).
+. From the *Tags* dropdown, search for `auto_disabled_8.0`.
+. Click *Select all _x_ rules*, or individually select the rules you want to re-enable.
+. Click *Bulk actions -> Enable* to re-enable the rules.
+
+Alternatively, you can use the <<bulk-actions-rules-api, Bulk rule actions>> API to re-enable rules.
+
+[float]
+[[fda-upgrade]]
+==== Full Disk Access (FDA) approval for {elastic-endpoint}
+
+When you manually install {elastic-endpoint}, you must approve a system extension, kernel extension, and enable Full Disk Access (FDA). There is a new FDA requirement in 8.x. Refer to <<elastic-endpoint-deploy-reqs>> to review the required permissions.
+
+[float]
+[[data-views-upgrade]]
+==== Requirements to display Data views in the {security-app}
+
+To make the *Data view* option appear in an environment with legacy alerts, a user with elevated role privileges must visit the {security-app}, open a page that displays alert data (such as the Overview page), then refresh the page. The user's role privileges must allow them to enable the detections feature in a {kib} space. Refer to <<enable-detections-ui, Enable and access detections>> for more information.
+
+NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page with alert data in {elastic-sec} will make the *Data view* option appear in the {elastic-sec} UI.
```

Comment:
This note was a bit confusing to me, I think you could remove it and change the parenthetical in the previous paragraph to say "...open a page that displays alert data (at least one alert must be present), then refresh the page." The current parenthetical that I'm suggesting we replace, "(such as the Overview page)" could probably be safely replaced since the Alerts page is so prominent in the nav menu.